PluginController/EntityPluginController.php

```diff
@@ -397,4 +389,10 @@ protected function doGetPaymentInstruction($id)
 
         return $paymentInstruction;
     }
+
+    protected function doRollback()
+    {
+        $this->entityManager->getConnection()->rollback();
+        $this->entityManager->close();
```

Comment:
This a really bad idea to close an entity manager.  If the exception related to database and comes from entity manager it will be already closed. 

The entity manager is only yours and could be used outside of jms payment bundle scope for some other stuff. Same http request may do some extra work.